config.json

```
@@ -120,12 +120,21 @@
       }
     ],
     "practice": [
+      {
+        "slug": "two-fer",
+        "name": "Two Fer",
+        "uuid": "1706ec00-9e51-45d1-ac3e-01b0360ea950",
+        "practices": ["strings", "basics"],
+        "prerequisites": ["strings", "basics"],
```

You don't strictly need the `basics` concept I think as that should be the very first concept that is required for any other concept to be unlocked. So `strings` can only be unlocked when `basics` was unlocked before. Not that it hurts to have `basics` listed here, mind.

```suggestion
        "practices": ["strings"],
        "prerequisites": ["strings"],
```